src/documents/tests/test_workflows.py

```diff
@@ -997,6 +998,39 @@ def test_document_updated_workflow(self):
 
         self.assertEqual(doc.custom_fields.all().count(), 1)
 
+    def test_document_updated_workflow_existing_custom_field(self):
```

Comment:
Maybe add the given/when/then if you can

Reply:
Sure!